README.adoc

```diff
@@ -0,0 +1,203 @@
+[id="readme"]
+= odo - Developer-focused CLI for OpenShift image:/docs/img/openshift.png[float="right",align="center"]
```

Comment:
So I found a fix, you can just add raw HTML to here in order to render this correctly..

See my branch here: https://github.com/cdrage/odo/tree/preview

Essentially, remove what you have now for image and at the top of the document add:

```
+++
<img align="right" role="right" width="180" src="/docs/img/openshift.png?raw=true"/>
+++
```

Reply:
I will try that @cdrage but I find the TOC not rendering in your branch too. Throughout my attempts, I have been able to either have the Image or the TOC, finally I reached what I see on this PR.
The TOC syntax is peculiar in that it needs to be right below the top most lines. As in there should be no line breaks between the content on the first line and the TOC syntax. Only then it renders correctly.

Reply:
Tried, image looks perfect, though slightly big. But TOC is gone.
See here for what I described above: http://discuss.asciidoctor.org/toc-not-working-td5427.html#a5428